pkg/api/latest/dynakube/otlpexporterconfiguration_props.go

```diff
@@ -0,0 +1,13 @@
+package dynakube
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/latest/dynakube/otlpexporterconfiguration"
+)
+
+func (dk *DynaKube) OTLPExporterConfiguration() *otlpexporterconfiguration.OTLPExporterConfiguration {
+	otlpExporterConfiguration := &otlpexporterconfiguration.OTLPExporterConfiguration{
```

Comment:
is any reason to duplicate package name and struct name?

can we try something like `otlpexporterconfiguration.Config`?

Reply:
Here I tried to use the same style as for the other config struct, e.g. like the [MetadataEnrichment](https://github.com/Dynatrace/dynatrace-operator/blob/main/pkg/api/latest/dynakube/metadataenrichment/spec.go#L17) - but I agree that the naming is a bit clunky here, so we can also rename this to what you suggested

Reply:
in this case, should we keep the current naming for now, as this is currently done in pretty much all sub packages of the Dynakube API? Otherwise, this PR would need to change all those as well if we want to keep consistent

Reply:
yes, @andriisoldatenko can you create a little techdept ticket to fix this? 👀

Reply:
on it 👮

Reply:
https://dt-rnd.atlassian.net/browse/DAQ-14905